metrics/stackdriver_exporter.go

```diff
@@ -354,7 +354,7 @@ func getStackdriverSecret(secretFetcher SecretFetcher) (*corev1.Secret, error) {
 			return nil, err
 		}
 
-		sec, secErr = kubeclient.CoreV1().Secrets(secretNamespace).Get(secretName, metav1.GetOptions{})
+		sec, secErr = kubeclient.CoreV1().Secrets(secretNamespace).Get(context.Background(), secretName, metav1.GetOptions{})
```

Comment:
Do we need to plumb the context through here?

Reply:
done in 77678bb83f220dd422bf886e7ce009f6f23d6616